src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Extern.java

```diff
@@ -590,7 +591,7 @@ private boolean isAutomaticModule(ModuleElement me) {
             return false;
         } else {
             ModuleSymbol msym = (ModuleSymbol) me;
-            return (msym.flags() & Flags.AUTOMATIC_MODULE) != 0;
+            return msym.isFlagSet(TypeSymbolFlags.AUTOMATIC_MODULE);
```

Comment:
The change is OK, but the code being changed is not. There should not be references to any `javac.code.*` types outside of the `WorkArounds` class.  Ideally, this function (`isAutomaticModule`) should be on the `Elements` class.

Reply:
FYI, PR 2537 will move this code to the `Workarounds` class.